Package.swift

```diff
@@ -2,21 +2,37 @@
 
 import PackageDescription
 
+#if compiler(>=5.9)
+/// TODO: Update to Lottie 4.4.0
+/// A precompiled XCFramework of the lottie-ios repo that was compiled with Xcode 15.2 / Swift 5.9.
+/// This XCFramework can be use by Xcode 15.0 and later.
+let lottieXCFramework = Target.binaryTarget(
+  name: "Lottie",
+  url: "https://github.com/airbnb/lottie-ios/releases/download/4.3.4/Lottie.xcframework.zip",
```

Comment:
The visionOS build will fail until we update this to a build of Lottie 4.4.0 (not released yet), but I tested this in CI with a custom XCFramework bundle built by Xcode 15.2 and confirmed that the current setup works:

<img width="1096" alt="image" src="https://github.com/airbnb/lottie-spm/assets/1811727/583b140b-a34a-4800-a34d-2194bee29165">